Ignore invisible widgets when hit testing (#4873)
Chatterino/chatterino2: CHANGELOG.md

- Bugfix: Fixed the Quick Switcher (CTRL+K) from sometimes showing up on the wrong window. (#4819)
- Bugfix: Fixed too much text being copied when copying chat messages. (#4812, #4830, #4839)
- Bugfix: Fixed empty page being added when showing out of bounds dialog. (#4849)

> - Bugfix: Fixed issue on Windows preventing the title bar to be dragged in the top left corner. (#4873)

Reviewer: ```suggestion
- Bugfix: Fixed issue on Windows preventing the title bar from being dragged in the top left corner. (#4873)
```
